src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -367,7 +413,14 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
     group: gist,
     links: ScrollablePromo,
     mpu: getMpuComponent(allowAdvertising),
-    wsoj: getWsojComponent,
+    wsoj: ({ data }: { data: Recommendation[] }) =>
+      getWsojComponent({
+        data,
+        blocks,
+        topStoriesContent,
+        featuresContent,
+        referrerVariant,
```

Comment:
Do we need to include `referrerExperimentName` here so that it isn’t `undefined` when Recommendations builds its Optimizely tracking?

Reply:
good spot. I have tried adding it but vscode won't let me commit due to gpg issues. I'll have to work out what is going on.

Reply:
ok restarting vscode seemed to help. added.